Core/Index.cs

```diff
@@ -1,11 +1,11 @@
 namespace Core
 {
-    public class Index
-    {
-        public decimal Value { get; set; }
-        public decimal Max30 { get; set; }
-        public decimal Min30 { get; set; }
-        public decimal Average30 { get; set; }
-        public decimal OldAverage30 { get; set; }
-    }
+	public class Index
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Replace all tab characters in this file by sequences of white-spaces.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=4931949)